solutions/usaco-917.mdx

```diff
@@ -11,12 +11,91 @@ author: Brad Ma
 **Time Complexity** $\mathcal{O}(N)$
 
 <LanguageSection>
+<CPPSection>
 
-<JavaSection>
+<Warning>
+Using `INT32_MAX` and `INT32_MIN` for the initial 
+bounds will cause integer overflow because the max
+and min will not necessarily execute first.
+</Warning>
+
+```cpp
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <string>
+#include <algorithm>
+
+using std::cout;
+using std::endl;
+using std::vector;
+
+// An arbitrary number that's large compared to the ones in the problem.
+const int LARGE = 1e9;
```

Comment:
umm, maybe `LARGE` is too vague of a word, how bout `MAX_V` or smth

Reply:
it isn't, `MAX_V` is 1000

Reply:
then why don't you just set the constant to `1001`?

Reply:
i feel like that would be too small

Reply:
errr, ok